packages/@okta/i18n/src/properties/login.properties

```diff
@@ -302,11 +302,13 @@ email.enroll.title = Set up Email Authentication
 email.enroll.description = Send a verification code to your registered email.
 email.link.terminal.msg = To finish signing in, return to the screen where you requested the email link.
 email.mfa.title = Verify with Email Authentication
+email.mfa.title.alternate = Verify with your email
```

Comment:
v1 uses prop from line #304, hence added a new one for IDX

Reply:
let's change to `oie.email.mfa.title`. fyi: https://github.com/okta/okta-signin-widget/pull/1215

Reply:
updated